auth0/src/test/java/com/auth0/android/provider/OAuthManagerTest.java

```diff
@@ -121,13 +104,23 @@ public void shouldHaveValidState() {
 
     @Test
     public void shouldHaveInvalidState() {
-        exception.expect(AuthenticationException.class);
-        OAuthManager.assertValidState("0987654321", "1234567890");
+        Assert.assertThrows(AuthenticationException.class, new ThrowingRunnable() {
```

Comment:
trying out the new way of asserting exceptions. The rule for "ExpectedException.none()" is deprecated